### ISSUE TITLE
AssertionError BUG

### ISSUE THREAD

**Issue by harrylee999** [datvuthanh/HybridNets#38]
if i just want to seg one class，such as seg_list only have ’road‘.
Then i run train.py ,
 in loss.py line 538,
 in soft_tversky_score assert output.size() == target.size()
 AssertionError

then i debug code,find output.size() = torch.Size([2, 1, 245760])   target.size() = torch.Size([2, 1, 491520])

How to fix that???

**Comment by xoiga123** [comment]
Should work now

**Comment by xoiga123** [comment]
Btw, I fixed this by using softmax for both binary and multi-class. While it is learnable, [some](https://stats.stackexchange.com/questions/207049/neural-network-for-binary-classification-use-1-or-2-output-neurons) advocate against this approach. I'll separate binary, multi-class and multi-label in #40.
